diff --git a/web/src/pages/Login.jsx b/web/src/pages/Login.jsx
index 5c56e30..d0b1ba4 100644
--- a/web/src/pages/Login.jsx
+++ b/web/src/pages/Login.jsx
@@ -37,7 +37,16 @@ function Login() {
       <div className="format lg:format-lg">
         <h2>Login to your account</h2>
       </div>
-      <GoogleLoginButton />
+
+      <div className="text-center">
+        <p className="text-lg font-bold">This is a demo of BookLogr</p>
+        <p>Some features are <a className="font-medium text-blue-600 dark:text-blue-500 hover:underline" href="https://github.com/mozzo1000/booklogr/demo">disabled.</a></p>
+        <br />
+        <p className="font-bold">To login use,</p>
+        <ul>Email: demo@booklogr.app</ul>
+        <ul>Password: demo</ul>
+      </div>
+
       <Card>
         <form className="flex flex-col gap-4" onSubmit={handleLogin}>
           <div>
